mautrix/bridge/matrix.py

```diff
@@ -445,9 +446,11 @@ async def handle_encrypted(self, evt: EncryptedEvent) -> None:
         except DecryptionError as e:
             self.log.warning(f"Failed to decrypt {evt.event_id}: {e}")
             self.log.trace("%s decryption traceback:", evt.event_id, exc_info=True)
+            self.send_decrypted_checkpoint(evt, e, True)
```

Comment:
I think you need to send a checkpoint in the `except SessionNotFound too`, because the first 5 second wait already happens inside `decrypt()`